src/popup/router/pages/TransactionDetails.vue

```diff
@@ -186,7 +186,8 @@ export default {
 @import '../../../styles/typography';
 
 .transaction-details {
-  overflow-x: hidden;
+  height: 100%;
```

Comment:
I can't get the reasoning, also I don't think that usual pages should set theit height explicitly. Would be nice to avoid extra scrolling of this page and others.